Rubberduck.VBEEditor/SafeComWrappers/VB/VbeFactory.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Rubberduck.VBEditor.SafeComWrappers.VB.Abstract;
+using VBAIA = Microsoft.Vbe.Interop;
+using VB6IA = Microsoft.VB6.Interop.VBIDE;
+
+namespace Rubberduck.VBEditor.SafeComWrappers.VB
+{
+    public static class VBEFactory
+    {
+        public static IVBE Create(object application)
+        {
+            switch (application)
+            {
+                case VBAIA.VBE vbae:
+                    return new VBA.VBE(vbae);   
+                    
+                case VB6IA.VBE vb6e:
```

Comment:
I think I'd prefer seeing `case VB6IA.VBE ide:` in both branches. Also I'll have to hold myself back very hard to not rename `VB6IA` to just `VB6` and `VBAIA` to `VBA`.

Reply:
changed to ide. WRT to the IA - at one point during construction I was working with both COM and Wrapper objects, and wanted the suffix to help disambiguate. But now the namespaces are sorted, it seems that's no longer an issue, so I'll remove the IA litter.

Reply:
Would `VBA.Interop` and `VB6.Interop` work?